config/config.ini

```diff
@@ -457,8 +451,6 @@ name = Daniel Roy Greenfeld
 [http://www.gefira.pl/blog/tag/planet-python/feed/]
 name = Dariusz Suchojad
 
-[http://datacommunitydc.org/blog/tag/python/feed/]
```

Comment:
Looks like this one has been changed to: http://www.datacommunitydc.org/blog?format=RSS&tag=python

Reply:
ok, I am going to remove this change but really strange because that was a 404 when I have tested this URL.